File: BE/Const/Enum/ReportStatus.cs
using System.ComponentModel;

namespace TruyenCV;

public enum ReportStatus
{
    [Description("Chờ xử lý")] Pending = 1,
    [Description("Đang xử lý")] InProgress = 2,
    [Description("Đã giải quyết")] Resolved = 3,
    [Description("Từ chối")] Rejected = 4
}

File: FE/src/const/enum/report-status.ts
export enum ReportStatus {
	Pending = 1,
	InProgress = 2,
	Resolved = 3,
	Rejected = 4,
}

export const ReportStatusLabel: Record<ReportStatus, string> = {
	[ReportStatus.Pending]: "Chờ xử lý",
	[ReportStatus.InProgress]: "Đang xử lý",
	[ReportStatus.Resolved]: "Đã giải quyết",
	[ReportStatus.Rejected]: "Từ chối",
};
